app/models/order.rb

```diff
@@ -1,6 +1,7 @@
 # frozen_string_literal: true
 
 class Order < ApplicationRecord
+  enum status: { not_delivered: 0, delivered: 1 }
```

Comment:
Personally I would use `boolean` instead of enum at this stage of development, since we have only two options - `delivered`/`not_delivered`. Let's leave it like it is - treat this comment as opinion, not suggestion :)

Reply:
I used the enum because there was a link to this in the materials for this sprint

Reply:
ah got it - I do not have the access to those materials yet :D